language/en/common.php

```diff
@@ -44,6 +44,7 @@
 	'IDEA_TOPIC'            => 'Idea topic',
 	'IDEA_POSTER'           => 'Posted by %s', // Warning: submitted to db
 	'IDEAS_HOME'			=> 'Ideas Home',
+	'IDEAS_NOT_AVAILABLE'	=> 'You must configure Ideas properly to do this.',
```

Comment:
Nah. Although this is really just an ext for phpBB, if another forum admin installs it, they can figure it out without telling the world they didn't configure it yet ;)